src/modules/add-task.js

```diff
@@ -0,0 +1,10 @@
+function addNewTask(todoTasks, description) {
+  const newTask = {
+    description,
+    completed: false,
+    index: todoTasks.length,
+  };
+  todoTasks.push(newTask);
+}
+
```

Comment:
- [ ] When I'm trying to add a new task, I'm getting the below error. Please make sure to fix this issue and users should be able to add tasks without any errors. Also, make sure other functionalities(remove and edit) also work as expected.

![image](https://user-images.githubusercontent.com/83803414/228835454-12c4584c-34d8-40e0-b586-3f2f7953390b.png)